### PR TITLE
Testing PR

### PR DESCRIPTION
**Pull request by gowrishankkar**

### CODE REVIEW

File: ig-reels-clone/src/VideoHeader.js [hunk 1]
```diff
@@ -2,6 +2,17 @@ import React from "react";
 import "./VideoHeader.css";
 import ArrowBackIcon from "@mui/icons-material/ArrowBack";
 import PhotoCameraIcon from "@mui/icons-material/PhotoCamera";
+
+
+
+
+
+
```
Comment (gowrishankkar) [inline ig-reels-clone/src/VideoHeader.js:10]: revert

Comment (Pavithrank02) [inline ig-reels-clone/src/VideoHeader.js:10]: done

File: ig-reels-clone/src/VideoHeader.js [hunk 2]
```diff
@@ -2,6 +2,17 @@ import React from "react";
...
+
+
+
 function VideoHeader() {
```
Comment (Pavithrank02) [inline ig-reels-clone/src/VideoHeader.js:16]: test

File: ig-reels-clone/src/VideoHeader.js (hunk 1 above)
Comment (gowrishankkar) [reply ig-reels-clone/src/VideoHeader.js:10]: this is for thus reasoon

**Pull request merged**